scripts/grapheme_properties.ecr

```diff
@@ -49,7 +49,7 @@ struct String::Grapheme
   # These ranges were taken from
   # http://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakProperty.txt
   # as well as
-  # http://www.unicode.org/Public/13.0.0/ucd/emoji/emoji-data.txt
+  # http://www.unicode.org/Public/UCD/latest/ucd/emoji/emoji-data.txt
```

Comment:
Shouldn't this better reference the exact version?

Reply:
The one in the docs for `String::Grapheme#break?` would require something like:

```crystal
{% begin %}
# ... https://www.unicode.org/Public/UCD/{{ Unicode::VERSION.id }}/ucd/auxiliary/GraphemeBreakTest.html ...
def self.break?(lbc : Property, tbc : Property) : Bool
  # ...
end
{% end %}
```